exercises/practice/leap/config.properties

```
@@ -0,0 +1,183 @@
+# Examples:
+# cobolcheck.injectedCodeTag.start = ###INJECT START###
+# cobolcheck.injectedCodeTag.end = ###INJECT END###
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.injectedCodeTag.start = null
+cobolcheck.injectedCodeTag.end = null
+
+#---------------------------------------------------------------------------------------------------------------------
+# A tag written at the start of entities stubbed by default. Recommended value-length <= 4.
+# Note: The tag will appear only when cobolcheck stubs lines by default.
+# This is the case for CALLs and batch file IO verbs.
+# Default is null, which will prevent the tag from appearing.
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.stub.comment.tag = null
+
+#---------------------------------------------------------------------------------------------------------------------
+# Determines if cobolcheck should generate code, such that decimal point is comma.
+# The default is "false". The value should be set to "true" if the compiler is set to
+# read decimal points as commas. If the cobol source program sets DECIMAL-POINT IS COMMA,
+# this configuration will be overwritten.
+# Example: 1,385,481.00 (decimalPointIsComma = false)
+# Example: 1.385.481,00 (decimalPointIsComma = true)
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.decimalPointIsComma = false
+
+#---------------------------------------------------------------------------------------------------------------------
+# If the source program contains rules as the first line follwed by CBL, the given value will be appended
+# to this.
+# If no CBL is found in source, it will be added along with the given value
+# default is null, which will make no changes.
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.append.rules = null
+
+#---------------------------------------------------------------------------------------------------------------------
+# Path for the generated Cobol test code
+# Default: ./
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.test.program.path = ./
+
+#---------------------------------------------------------------------------------------------------------------------
+# Suffix to append to the name of each program under test to produce the name of the corresponding
+# test program that contains the merged test code.
+# Example: For program ABCXYZ4, if suffix is T.CBL then the test program name will be ABCXYZ4T.CBL.
+#---------------------------------------------------------------------------------------------------------------------
+cobolcheck.test.program.name = TEST.COB
```

At least the file extension is expected to be in lower case by the test scripts.
```suggestion
cobolcheck.test.program.name = test.cob
```